### PR TITLE
PR : Quick sort in C programming

### PR DESCRIPTION
**Pull request by Subhashini014**
I am raising pull request for the instructions you mentioned in my before PR I did more commits. So I am doing again now without any conflicts to base branch.

### CODE REVIEW

File: C-ProgramQuicksort/Quicksort.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+Definition:
```
Comment (Abishethvarman) [inline C-ProgramQuicksort/Quicksort.md:1]: make this as heading

File: C-ProgramQuicksort/Quicksort.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+Quick Sort is a sorting algorithm that follows the divide-and-conquer strategy to efficiently sort an array or list of elements. It works by selecting a "pivot" element from the array and then rearranging the other elements into two sub-arrays – one with elements less than the pivot and another with elements greater than the pivot. This process is performed recursively until the entire array is sorted.
+
+Explanation in Simple Terms:
```
Comment (Abishethvarman) [inline C-ProgramQuicksort/Quicksort.md:5]: heading

File: C-ProgramQuicksort/Quicksort.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+Recursion:
+
+The same partitioning process is applied to the sub-arrays created on the left and right of the pivot. This continues until the sub-arrays are so small that they don't need further partitioning.
+
+Combining: 
+
+The sorted sub-arrays are combined to get the final sorted array.
+
+Code-Related Explanation:
```
Comment (Abishethvarman) [inline C-ProgramQuicksort/Quicksort.md:23]: heading

File: C-ProgramQuicksort/Quicksort.c [hunk 4]
```diff
@@ -0,0 +1,56 @@
+#include <stdio.h>
```
Comment (Abishethvarman) [inline C-ProgramQuicksort/Quicksort.c:1]: no need of separate .c file. Add that in the .MD file

File: C-ProgramQuicksort/Quicksort.md (hunk 1 above)
Comment (Abishethvarman) [inline C-ProgramQuicksort/Quicksort.md:1]: Make a h1 heading "Quick Sort in C programming" in the top of the readme

**Review by Abishethvarman — CHANGES_REQUESTED** [review]

**Comment by Abishethvarman** [comment]
Rename the PR title as " PR : Quick sort in  C programming"

**Comment by Subhashini014** [comment]
As per your request the changes are done.

**Comment by Abishethvarman** [comment]
> I am raising pull request for the instructions you mentioned in my before PR I did more commits. So I am doing again now without any conflicts to base branch.

also mention what algorithm and what language that you have implemented.

**Review by Abishethvarman — APPROVED** [review]
Approved to merge, @Thevapriyan07 check the description and merge it with correct labels

**Review by Thevapriyan07 — APPROVED** [review]
Seems good

**Comment by Subhashini014** [comment]
In my Hactoberfest account it is showing as not accepted after you merged my pull request.

**Comment by Abishethvarman** [comment]
> In my Hactoberfest account it is showing as not accepted after you merged my pull request.

refresh in 15 mins

**Pull request merged**